File: pulumi-infra-project/pulumi/examples/typescript/digitalocean_spaces_example.ts
// Pulumi DigitalOcean Spaces Example (TypeScript)
import * as pulumi from "@pulumi/pulumi";
import * as digitalocean from "@pulumi/digitalocean";

const space = new digitalocean.SpacesBucket("example-space", {
    region: "nyc3"
});

export const bucketName = space.name;

File: pulumi/examples/python/digitalocean_spaces_example.py
"""
Pulumi DigitalOcean Spaces Bucket Example (Python)

Setup:
  pip install pulumi pulumi_digitalocean
Usage:
  pulumi up
"""
import pulumi
import pulumi_digitalocean as digitalocean

bucket = do.SpacesBucket('example-space',
    region='nyc3'
)
example_space = digitalocean.SpacesBucket(
  "example-space",
  region="nyc3"
)

pulumi.export("bucket_name", example_space.name)
